src/mlui/enums/Adabound.py

```diff
@@ -0,0 +1,225 @@
+
+
+
+import tensorflow.compat.v2 as tf
+from keras import backend
+from keras.optimizers import Optimizer
+
+class AdaBound(Optimizer):
+    """
+    Adabound optimizer implementation 
+    # References
+        - [Adaptive Gradient Methods with Dynamic Bound of Learning Rate]
+          (https://openreview.net/forum?id=Bkg3g2R9FX)
+
+
+
+    Parameters
+    -----------
+        learning_rate : float 
+            Adam learning rate (default: 1e-3)
```

Comment:
The _standard_ way of writing default values in NumPy docstyle is to place them after the type specification. You can refer to the [Parameters](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) section for the correct format. You should also use a dot at the end of the description.

Reply:
solved